### PR TITLE
feat(ui): New Colors page in Storybook

### PR DESCRIPTION
**Pull request by vuluongj20**
Adding a new Colors page to Storybook, along with links to it in _Getting Started_ and _Changelog_.

Screen Shot 2021-11-09 at 2 46 42 PM

Screen Shot 2021-11-09 at 2 46 55 PM

### CODE REVIEW

File: docs-ui/components/sample.tsx [hunk 1]
```diff
@@ -1,12 +1,89 @@
...
+   * switching the local theme context between
+   * light and dark mode. Useful for previewing
+   * components in both modes.
+   */
+  showThemeSwitcher?: boolean;
+  /** Remove the outer border and padding */
+  noBorder?: boolean;
+};
+
+/** Expose the selected theme to children of <Sample /> */
```
Comment (evanpurkhiser) [inline docs-ui/components/sample.tsx:24]: Use 

```
/**
 * comment here
 */
```

over this style. There's no eslint rule for it :(

File: docs-ui/stories/core/colors/tables.tsx [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+  {
+    id: 'gray500',
+    colors: [
+      {
+        name: 'Gray 500',
+        lightValue: '#2B2233',
+        darkValue: '#EBE6EF',
+      },
+    ],
+  },
```
Comment (billyvg) [inline docs-ui/stories/core/colors/tables.tsx:52]: Will you be updating these later to source them from theme?

Comment (vuluongj20) [reply docs-ui/stories/core/colors/tables.tsx:52]: Yepp! Will update these once #29917 is approved and merged.

File: docs-ui/components/sample.tsx [hunk 3]
```diff
@@ -17,11 +17,14 @@ type Props = {
    * components in both modes.
    */
   showThemeSwitcher?: boolean;
-  /** Remove the outer border and padding */
+  /**
+   * Remove the outer border and padding
+   */
   noBorder?: boolean;
-};
```
Comment (billyvg) [inline docs-ui/components/sample.tsx:22]: this got removed?

**Review by billyvg — APPROVED** [review]

**Comment by billyvg** [comment]
Looks good aside from the removed braces

**Pull request merged**